modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -126,7 +129,7 @@ public ObjectEntry getByExternalReferenceCode(String externalReferenceCode)
 
 	@Override
 	public EntityModel getEntityModel(MultivaluedMap multivaluedMap) {
-		if (_entityModel == null) {
+		if ((_entityModel == null) || !_isEntityModelUpdated()) {
```

Comment:
@p-albuquerque As a matter of simplicity, we should remove this verification and always build the EntityModel for now...

Verifying if there's a change every time can be costly, you know?